### DISCUSSION TITLE
Nordigen reduce free requests on API from 1 Juni 2023

### DISCUSSION THREAD

**Discussion by poudenes** [firefly-iii/firefly-iii#7297] [General]
Here the email I received:

Dear user,

As you may be aware, Nordigen was acquired by [GoCardless](https://nordigen.cmail20.com/t/t-l-fkkjldy-jiururitui-y/), a leading provider of bank payments, in July 2022. We’re working hard to integrate both organisations so that we can provide both open banking data and payments.  As part of the integration process we’d like to let you know about some important changes that we’re making to the provision of our Account Information API. 

**Introducing our new Pay As You Go plan**
We’re continuing to provide free access to our Account Information API for up to 50  active end-user data requisitions* per month. If you don’t exceed this amount your access will remain free of charge. 

From 1 June 2023 free usage will be capped at 50 active requisitions per month and our new Pay As You Go plan will come into effect. If you wish to generate more than 50 active requisitions per month you will be asked to upgrade to the Pay As You Go plan.

If you opt for the Pay As You Go plan you will be charged a flat monthly fee of 100 EUR plus 0.25 EUR per active requisition per month. The plan includes access to:

- Standard customer support, with prioritisation for your support requests 
- UI configuration capabilities
- Entity verification

Please note that from 1 June 2023, should you wish to continue using UI configuration with your customised consent UI, or to remove the verification banner from the consent UI, you will be required to upgrade to our Pay As You Go plan.
Example fees are outlined below:

![Screenshot2023-03-31at10 46 35-9900000000079e3c](https://user-images.githubusercontent.com/4399532/229087863-ea644c6f-0dbc-45fd-892e-e2ace0b96c52.png)

**What does this mean for you?**
You can continue to access account information for free for up to 50 active end-user data requisitions per month. You do not need to make any changes to your account and you can continue to access our Account Information API as usual.

We’ll notify you if you reach the free usage limit. When you do, you’ll be unable to create any new requisitions until the following month. If you want to upgrade to the Pay As You Go plan now please contact [sales@nordigen.com](mailto:sales@nordigen.com) or your account manager directly.

If you continue to remain on the free plan, from 1 June 2023 the UI configuration and entity verification settings will revert to default settings, namely your access to these features will be disabled.

If you have questions about the upcoming changes, please let us know by replying to this email.

Kind regards,
Nordigen - a GoCardless company

**Comment by JC5** [comment]
Thanks for sharing. I got it as well. 

As I understand, this should only influence Firefly III users that import from more than 50 banks. So effectively, nobody?

**Comment by poudenes** [comment]
50  active end-user data requisitions* per month.

For me is this 50 requests a month. I have 2 banks check every day. So will be around 62 requests in a month.

Thats how I read this.

**Reply by JC5** [reply]
I don't.

> *A requisition links the client company (you) with the end-user's consent to access their account(s) available under the same login at one specific bank. All requisitions where account data has been available for any amount of time during the month will be counted towards the use number/limit. 

To me, this limits the number of active connections, not the amount of times you use it. Since you have 2 banks, you have 2 requisitions.

**Reply by piotrcichosz** [reply]
> Since you have 2 banks, you have 2 requisitions.

Unless for next imports, instead of importing previously downloaded configuration file (which contains requisition id) create new (so every time log into bank to grant access to nordigen) - then each time granting access in bank is new active requisition 
I did that by mistake before and have few active requisitions for 1 bank ;)

**Comment by poudenes** [comment]
Asked for some clarification:

Hello Peter, 
 
Thank you for your question!
 
It doesn't necessarily have to be 50 different banks, but rather 50 active connections (requisitions). 
 
Let's say you have 50 clients, whose bank accounts you would like to access. 
So you create 50 new connections and let your clients link their accounts. 
If all of them manage to link their accounts successfully, in this case, it will be 50 active connections, and the new pricing will be applied. 
Even if all of the clients are from the same bank, it will still be considered as 50 different active connections. 
 
Hope this clarifies!
 
If you have any further questions, feel free to reach out.

**Reply by Dinth** [reply]
Hi @poudenes.
I am a new Firefly III user and just set up the sync three days ago.
1. It seems that with United Kingdom Capital One card i need to log in every time i am importing data (importing data from Firefly III config file doesnt work) - will that count to my requisitions?
2. When ive been setting up my United Kingdom American Express i found a weird bug - the set up literally worked after 10+ attempts to log in via Nordigen. I guess each login creted a requisition, but how can i now remove the ones which dont work?

**Comment by poudenes** [comment]
I have 2 banks in my formal Nordigen Account. Now I also see my usage 6/50. Every time when I check 1 bank the X will change. So when I use 2 banks a day that will be 60 or 62 times. (depend how many days in a month)

So the discussion is clear now. It's not how many connections you use. But how many banks you use.
That means for me I have to move from every day check to every 2 day check. So I will keep my usage below 50 every month

<img width="600" alt="Screenshot 2023-06-19 at 13 20 21" src="https://github.com/firefly-iii/firefly-iii/assets/4399532/a8505870-93af-42f2-852b-6bfeaf033b03">

**Reply by imjuzcy** [reply]
Do you log in to the bank every time you check? If that's the case, that's why your active requisitions increases every check. For me, I log in to my bank account once for the first import, then I download the config file for the importer, and for subsequent checks I upload the downloaded config file. I check every four hours, so 6 times a day, but my active requisition count does not increase.

**Comment by poudenes** [comment]
One bank is using the config file also. My other back has problems to connect. They investigate it now. Maybe that was reason of increasing so fast. I tried many times to link my other bank so I have a new config file.
